CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set(CARTOGRAPHER_SOVERSION ${CARTOGRAPHER_MAJOR_VERSION}.${CARTOGRAPHER_MINOR_VE
 
 include("${CMAKE_SOURCE_DIR}/cmake/functions.cmake")
 google_initialize_cartographer_project()
```

Comment:
This should probably be renamed to google_initialize_project() since it applies to more projects now. Can wait for another PR, though.